anaconda.spec.in

```diff
@@ -176,6 +176,9 @@ for live installations.
 Summary: Installation environment specific dependencies
 Requires: udisks2-iscsi
 Requires: libblockdev-plugins-all >= %{libblockdevver}
+%if ! 0%{?rhel}
+Requires: libblockdev-lvm-dbus
```

Comment:
I would like to point out that we also have `install-img-deps` specifically for the boot.iso dependencies, while `install-env-deps` is used in all our environments (for example, Live CD). However, in this case it seems fine to keep it here.

Reply:
So `install-env-deps` applies for boot.iso too? If yes, I think having it in `install-env-deps` makes more sense.

Reply:
Yes :-)